### DISCUSSION TITLE
Proper JSON-API Support

### DISCUSSION THREAD

**Discussion by manosbatsis** [refinedev/refine#5173] [Integration request] [answered]
Elide 7 supports the [JSON-API](https://jsonapi.org/) spec. I've modified [refine-elide-rest](https://github.com/chirdeeptomar/refine-elide-rest) a bit to make it work but it's both outdated and basic. 

Are there any relevant/helpful  integrations out there? I'm looking at how to:

- Abstract the structural differences of JSON-API vs traditional REST data as much as possible.
- Ideally utilize the additional info available: links, relationships etc.
- Provide a contribution focused on the above using a data provider and other components as applicable (inferencer?) etc.

**Comment by omeraplak** [comment, marked as answer]
Hey @manosbatsis ,
Thank you for contacting us! A data provider for JSON:API was recently released by the community. Could this be useful for you? https://github.com/mahirmahdi/refine-jsonapi

**Reply by ghbishal** [reply]
@omeraplak are you planning to create a examples for json api?

**Comment by manosbatsis** [comment]
Thanks a bunch @omeraplak, will try it out!
